.github/workflows/main.yml

```diff
@@ -0,0 +1,71 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Build pdf
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the main branch
+on:
+  push:
+    branches: [ main ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      # Runs a single command using the runners shell
+      - name: cd on workspace
+        run: cd $GITHUB_WORKSPACE
+
+
+      - name: Github Action for LaTeX
+        # You may pin to the exact commit or the version.
+        # uses: dante-ev/latex-action@259e5ff953ad4d4616fbd09cc87a526ee5c4499a
+        uses: xu-cheng/latex-action@v2
+        with:
+          # The root LaTeX file to be compiled
+          root_file: sample.tex
+          # The working directory for the latex compiler to be invoked
+          working_directory: . # optional
+          # LaTeX engine to be used
+          # compiler: pdflatex # optional, default is latexmk
+          # Extra arguments to be passed to the latex compiler
+          # args: ""
+          # Install extra packages by apt-get
+          # extra_system_packages: # optional
+
+                        
+      - name: Rename pdf
+        run: mv sample.pdf cv.pdf
+      
+      - name: Create empty changelog
+        run: echo "# Update CV" > CHANGELOG.md
+        
+      - name: Update Release
```

Comment:
It seems to me that it is better to separate this part so that it runs only in the master branch, for example, by condition
`if: github.ref == 'refs/heads/master'`
https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idif

Also, it would be nice to use `actions/upload-artifact` and `actions/download-artifact`, so that we can see which file is still on the pull request, and in the wizard, download the artifacts and adding in to release

To add assets to the release, we can also use `actions/upload-release-asset` or `svenstaro/upload-release-action`

Reply:
OOO
NICE! 

I definitely believe this is the good way to follow.
I should spend time on it!!!